Dockerfile

```diff
@@ -0,0 +1,69 @@
+#
+# Dockerfile for building coordinator node images
+#
+
+FROM openjdk:8u312-jre-slim as base
+
+RUN apt update -qq \
+    && apt install iproute2 libaio1 -y \
+    && apt autoremove --yes \
+    && apt clean all \
+    && rm -rf /var/lib/apt/lists/*
+
+#RUN apt update -qq \
+#    && apt-get upgrade -y \
+#    && apt install iproute2 libaio1 -y \
+#    && apt autoremove --yes \
+#    && apt clean all \
+#    && rm -rf /var/lib/{apt,dpkg,cache,log}/
+
+# CQL
+EXPOSE 9042
+
+# GraphQL
+EXPOSE 8080
+
+# Auth
+EXPOSE 8081
+
+# Health
+EXPOSE 8084
+
+# this will include all persistence jars, we'll remove the ones we don't want below
```

Comment:
Would this still leave those jars as part of the image (or rather, use up space)? I recall Docker image construction is sometimes tricky as snapshot is taken between steps.